builds/e2e/connectivity.yaml

```diff
@@ -105,11 +105,25 @@ jobs:
         condition: and(succeeded(), eq(variables['run.flag'], 1))
       - task: AzureKeyVault@1
         condition: eq(variables['run.flag'], 1)
-        displayName: 'Azure Key Vault'
+        displayName: 'Azure Key Vault: EdgeBuildkv'
+        inputs:
+          azureSubscription: $(azure.subscription)
+          KeyVaultName: 'edgebuildkv'
```

Comment:
By hard-coding this key vault name, we're preventing people outside the product team from running end-to-end tests. So if we're going to support one key vault for IoT Hub secrets and another key vault for everything else, then the "everything else" key vault name needs to have a variable too.

Reply:
My understanding was that the purpose of this PR is to"
1. Allow someone on the product team to "bring your own hub resource to the table".
2. Shard the credentials of test resources into labeled keyvaults for more understandable secret-groupings

I never thought people outside of the product team would run our end to end tests, but it is something that makes sense. More work is probably required in other places too to get this to work.

Reply:
This was a goal in the beginning, but it wouldn't surprise me if, as the project grew and things happened fast, it fell between the cracks. This is a public project so there really shouldn't be anything here that isn't relevant to the public.